.travis.yml

```diff
@@ -24,8 +24,10 @@ env:
   - VERSION="8.0" TESTS="0" LINT_CHECK="0"
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="ubar_custom,astikar_custom,galaxia_custom,rockbotic_custom,rockbotic_website_crm,pesa_custom"
-  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="ubar_custom,astikar_custom,galaxia_custom,rockbotic_custom,rockbotic_website_crm,pesa_custom"
+  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="ubar_custom,astikar_custom,galaxia_custom,rockbotic_custom,rockbotic_website_crm,pesa_custom,hispaled_custom"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="ubar_custom,astikar_custom,galaxia_custom,rockbotic_custom,rockbotic_website_crm,pesa_custom,hispaled_custom"
+  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="ubar_custom,astikar_custom,galaxia_custom,rockbotic_custom,rockbotic_website_crm,pesa_custom,tumaker_custom"
```

Comment:
Por que el tumaker_custom no se añade directamente a los excludes que ya existen?

Reply:
@Daniel-CA ... el tumaker_custom se ha excluido 2 veces para ODOO y para OCA, para que no se ejecuten los test correspondientes de este módulo, pero también he tenido que crear 2 test nuevos para excluir "hisp...._custom", ya que por el formulario de pedidos de venta, se han hecho incompatibles.

Reply:
Lo que tendrías que hacer entonces es poner el de hispaled en el exclude y poner un include con solo el módulo hispaled.
Estos cambios antes de hacerlos revisa la documentación de que hace cada linea, ya que cuanto mas complicada sean las reglas del test, más aumenta su tiempo de ejecución, y lo que queremos evitar es que nos de el tan temido error 137 de timeout.
https://docs.travis-ci.com/user/customizing-the-build#build-timeouts

Reply:
@Daniel-CA ... cambiado el yml como he has dicho, Eskerrik asko.